prover/lang/smt-lang.md

```diff
@@ -226,19 +226,22 @@ module SMTLIB2-HELPERS
   rule SMTLIB2ScriptToString( COMMAND SCRIPT )
     => SMTLIB2CommandToString( COMMAND ) +String "\n" +String SMTLIB2ScriptToString( SCRIPT )
 
-  syntax CheckSATResult ::= CheckSATHelper(/*Query*/String, String/*Command*/)                                 [function]
-  syntax CheckSATResult ::= "CheckSAT.doWrite"  "(" /*Query*/String "," /*Command*/String "," IOFile ")"       [function]
-  syntax CheckSATResult ::= "CheckSAT.doClose"  "(" /*Query*/String "," /*Command*/String "," IOFile "," K ")" [function]
-  syntax CheckSATResult ::= "CheckSAT.doSystem" "(" /*Query*/String "," /*Command*/String "," String "," K ")" [function]
-  rule CheckSATHelper(Q, C)
-    => CheckSAT.doWrite(Q, C, #mkstemp("query-XXXXXX.smt"))
-  rule CheckSAT.doWrite(Q, C, #tempFile(FN, FD))
-    => CheckSAT.doClose(Q, C, #tempFile(FN, FD), #write(FD, Q))
-  rule CheckSAT.doClose(Q, C, #tempFile(FN, FD), .K)
-    => CheckSAT.doSystem(Q, C, FN, #close(FD))
-  rule CheckSAT.doSystem(Q, C, FN, .K) => CheckSAT.parseResult(#system(C +String FN))
-
-  rule CheckSAT.doWrite(_, C, E:IOError) => #error(E)
+  syntax CheckSATResult ::= CheckSATHelper(/*Prelude*/ String, /*Query*/String, String/*Command*/)                                 [function]
```

Comment:
Since we now target just one solver, we no longer need the "Command" parameter. Could you also move this function and the `K-IO` import to the `CVC4` module.